### PR TITLE
Add more ctx to error message

### LINKED ISSUE #192: Add more context to Failed to Install error
It happens from time to time that benchmarks fail and there are endless messages of:

```
I 2020-03-19T18:24:12.254106156Z 2020-03-19 18:24:12.253 [] [raft-server-1-raft-partition-partition-2] WARN  io.atomix.protocols.raft.roles.LeaderAppender - RaftServer{raft-partition-partition-2} - Failed to install 0
 
I 2020-03-19T18:24:12.274864880Z 2020-03-19 18:24:12.274 [] [raft-server-1-raft-partition-partition-2] WARN  io.atomix.protocols.raft.roles.LeaderAppender - RaftServer{raft-partition-partition-2} - Failed to install 0
 
I 2020-03-19T18:24:12.300388818Z 2020-03-19 18:24:12.300 [] [raft-server-1-raft-partition-partition-2] WARN  io.atomix.protocols.raft.roles.LeaderAppender - RaftServer{raft-partition-partition-2} - Failed to install 0
 
  undefined
```

Which is not really meaningful or helpful. We should try to improve the message so we are able to spot the problem better.

### PR DESCRIPTION
**Pull request by ChrisKujawa**
closes #192

### CODE REVIEW

File: protocols/raft/src/main/java/io/atomix/protocols/raft/roles/AbstractAppender.java [hunk 1]
```diff
@@ -220,7 +220,8 @@ protected void failAttempt(RaftMemberContext member, RaftRequest request, Throwa
     // when attempting to send entries to down followers.
     final int failures = member.incrementFailureCount();
     if (failures <= 3 || failures % 100 == 0) {
-      log.info("{} to {} failed: {}", request, member.getMember().memberId(), error.getMessage());
+      log.warn(
+          "{} to {} failed: {}", request, member.getMember().memberId(), error.getMessage(), error);
```
Comment (deepthidevaki) [inline protocols/raft/src/main/java/io/atomix/protocols/raft/roles/AbstractAppender.java]: Do we need both `error.getMessage()` and `error`?

Comment (ChrisKujawa) [reply protocols/raft/src/main/java/io/atomix/protocols/raft/roles/AbstractAppender.java]: Probably not. Just wanted to see also stacktrace etc. I dont like printing only the message. I can remove the message thing

Comment (deepthidevaki) [reply protocols/raft/src/main/java/io/atomix/protocols/raft/roles/AbstractAppender.java]: :+1:

Comment (ChrisKujawa) [reply protocols/raft/src/main/java/io/atomix/protocols/raft/roles/AbstractAppender.java]: updated branch

**Review by deepthidevaki — APPROVED** [review]

**Pull request merged**